### PR TITLE
NFC: Cleanups

### PR DESCRIPTION
**Pull request by ob**
This PR standardizes the C formatting using clang-format, and fixes a bunch of errors flagged by buildifier. It also removes some dead code. It is a non-functional change.

I was finishing  #52 but kept mixing up formatting changes with the actual changes so I figured I'd do the cleanups first and then rebase #52 on top of this.

### CODE REVIEW

File: rules/framework.bzl [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+
 load("@bazel_skylib//lib:paths.bzl", "paths")
-load("@bazel_skylib//lib:types.bzl", "types")
-load("@build_bazel_rules_swift//swift:swift.bzl", "SwiftInfo")
-load("@build_bazel_rules_swift//swift:swift.bzl", "swift_common")
-load("//rules:library.bzl", "PrivateHeaders", "apple_library")
-load("//rules/vfs_overlay:vfs_overlay.bzl", "VFSOverlay")
+load("@build_bazel_rules_swift//swift:swift.bzl", "SwiftInfo", "swift_common")
+load("//rules:library.bzl", "PrivateHeadersInfo", "apple_library")
+#load("//rules/vfs_overlay:vfs_overlay.bzl", "VFSOverlay")
```
Comment (segiddins) [inline rules/framework.bzl:6]: commented code

File: rules/library.bzl [hunk 2]
```diff
@@ -250,14 +252,18 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
...
+                                a `PrivateHeadersInfo` provider.
         namespace_is_module_name: Whether the module name should be used as the
                                   namespace for header imports, instead of the target name.
         default_xcconfig_name: The name of a default xcconfig to be applied to this target.
         xcconfig: A dictionary of Xcode build settings to be applied to this target in the
                   form of different `copt` attributes.
+        **kwargs: keyword arguments.
+
+    Returns:
+        Struct with a bunch of info
```
Comment (segiddins) [inline rules/library.bzl:264]: ha, nice

File: rules/framework.bzl (hunk 1 above)
Comment (ob) [reply rules/framework.bzl]: removed

File: rules/library.bzl (hunk 2 above)
Comment (ob) [reply rules/library.bzl:264]: 😂

**Comment by gyfelton** [comment]
conflicts due to my changes in previous PR. sorry :)

**Comment by ob** [comment]
No worries, it wasn't hard to merge.

**Pull request merged**